### DISCUSSION TITLE
Support for adding custom tags - grafana data source

### DISCUSSION THREAD

**Discussion by kaushiksrinivas** [grafana/grafana#50643] [Data Sources]
Option to add tags to grafana dashboard is present, for ex : "tags": [ "templated" ], as part of the dashboard json fields.
No such options are documented for grafana datasource.

There is a need for identifying the data sources added by a specific user/application and run GETALL datasources belonging to a user/application via custom tags. Since there is no documented option for querying the datasources based on tags, is there any other work around ways for adding custom labels/tags to datasource and getAll by these identifiers ?

Also, is there a plan to add support for custom tags for datasource ?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
